types/ember__application/index.d.ts

```diff
@@ -14,6 +14,13 @@ import { Router } from '@ember/routing';
 import Registry from '@ember/application/-private/registry';
 import Resolver from 'ember-resolver';
 import { AnyFn } from 'ember/-private/type-utils';
+import Owner from '@ember/owner';
+import type GlimmerComponent from '@glimmer/component';
+import EmberObject from '@ember/object';
+
+// Shut off default exporting; we don't want anything but the *intended*
+// public API present.
+export {};
```

Comment:
I understand the reason of this being here, but you can impemente differently  (as union) and get rid of additional type symbol (kiss)